### ISSUE TITLE
bug in linenr

### ISSUE THREAD

**Issue by joachimheintz** [csound/csound#978]
linenr does not produce the fade-out section any more.
this should give one second fade-in, one second sustain, and one second fade-out:

instr Sinus
 aSinus poscil .2, 400
 aOut linenr aSinus, 1, 1, .01
 out aOut, aOut
endin
schedule "Sinus", 0, 2

but the fade-out is not performed.

this is with csound from today's git.

**Comment by vlazzarini** [comment]
Just tested this with the latest
```
--Csound version 6.12 beta (double samples) Jul 12 2018
[commit: 39a8ba2922021148735fca8491dbece9b6bf7b4b]
```

and I cannot reproduce your issue. There is an attack of 1 sec, 1 sec sustain and 1 sec decay (exponential) for a total of 3 secs.  I also tested with 6.11 and the output is identical.

This on MacOs.

**Comment by joachimheintz** [comment]
can you try with ksmps=1?

i see now that it only occurs with this constellation.

**Comment by vlazzarini** [comment]
yes, that's happening with ksmps = 1. It happens on 6.11 as well, so the bug must have been there for a while.

**Comment by vlazzarini** [comment]
pushed a fix to git develop/
